vc.adoc

```diff
@@ -1076,6 +1087,163 @@ Content-Type: application/json;charset=UTF-8
+
+[[_request_headers_group_exp]]
+==== link:#_request_headers_group_exp[Request headers]
+
+[width="100%",cols="50%,50%",options="header",]
+|===
+|Name |Value
+|Accept |application/json
+|Content-Type |application/json
+|authorization |apiKey xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+|===
+
+[[_path_parameters_group_exp]]
+==== link:#_path_parameters_group_exp[Path parameters]
+
+[width="100%",cols="25%,25%,25%,25%",options="header",]
+|===
+|Parameter |Type |Optional |Description
+|provider |String |false |Unique ID of the Content Provider.
+|===
+
+[[_query_parameters_group_exp]]
+==== link:#_query_parameters_group_exp[Query parameters]
+
+[width="100%",cols="25%,25%,25%,25%",options="header",]
+|===
+|Parameter |Type |Optional |Description
+|exerciserUuid |String |false |Exerciser Unique ID.
+|===
+
+[[_request_fields_group_exp]]
+==== link:#_request_fields_group_exp[Request fields]
+
+[width="100%",cols="25%,25%,25%,25%",options="header",]
+|===
+|Parameter |Type |Optional |Description
+|body |String |false |GroupExp objects in JSON format.
+|===
+Example:
+----
+{
+   "provider":"GroupExp v1",
+   "data":[
+      {
+         "id":"9204039012020",
+         "name":"Step Express",
+         "activity":"Step Express"
+      },
+      {
+         "id":"9204039012035",
+         "name":"Test",
+         "activity":"Test"
+      }
+   ]
+}
+----
+
+[[_response_fields_group_exp]]
+==== link:#_response_fields_app_settings[Response fields]
+
+[width="100%",cols="25%,25%,25%,25%",options="header",]
+|===
+| Path | Type | Optional | Description
+| items | Map | false | Array of Key-Value to represent assets.
+| items[].groupexp_id | String | false | GroupExp object id.
+| items[].images | Array | false | Array of image objects.
+| items[].images[].format | String | false | Image style of the image file.
+| items[].images[].sourceUrl | String | false | Image object url of the image file.
+| items[].video |Array |true |Array with Videos brief info.
+| items[].video[].name |String |false |Name of the Video (e.g. 'RPM #79 Express', 'BODYPUMP #100').
+| items[].video[].id |String |false |Unique ID of the Video.
+| items[].video[].duration |Integer |false |Video duration in seconds.
+| items[].video[].episode |Object |true |Video Episode.
+| items[].video[].episode.number |Integer |true |Number of episode in season.
+| items[].video[].episode.season |Integer |true |Number of season.
+| items[].video[].thumbnail |String |false |Thumbnail of the Video.
+| items[].video[].customInfo |Map |false |Array of Key-Value to represent additional information of an entity. Partners could contain diverse information of same entities. This type designed to store that information.
+|===
+
+[[_example_request_group_exp]]
+==== link:#_example_request_group_exp[Example request]
+
+[source,highlightjs,highlight]
+----
+$ curl 'http://localhost:8080/api/groupexp/v1.0/content-providers/wichita/get-category-asset?exerciserUuid=8965a460-a79e-4bf7-b66c-7e34d8c34760' -i -X POST \
```

Comment:
Please replace `groupexp`  with `groupxclasses` or remove.

Reply:
And I would like to replace it with `GET /api/groupxclasses/v1.0/content-providers/{provider}/get-category-asset?source={source}&id={id}&name={name}&activity={activity}`

Reply:
fixed